command/utils.go

```diff
@@ -14,11 +18,68 @@ func (kw *stdoutkw) Write(k string) (err error) {
 	return err
 }
 
-func HandleClientError(err error) error {
-	if aerr, ok := err.(payload.APIError); ok {
-		if len(aerr.APIError.Fields) > 0 && aerr.Response.StatusCode == 403 {
-			return fmt.Errorf("Validation error: %v", aerr.APIError.Fields)
+func HandleClientError(err *client.APIError, verbose bool) error {
+	if perr, ok := err.Err.(*payload.Error); ok {
+		var errString string
+		// create error message according to response code
+		switch err.Response.StatusCode {
+		case 422:
+			if len(perr.Fields) > 0 {
+				errString = fmt.Sprintf("validation error: %v", perr.Fields)
+			}
 		}
+		// use default server error
+		if errString == "" {
+			errString = perr.Error()
+		}
+		errString = "server side error: " + errString
+		if verbose {
+			errString += verboseClientError(err)
+		}
+		return errors.New(errString)
+	} else {
+		if err != nil && verbose {
+			return errors.Wrap(err.Err, "debug info: \n"+verboseClientError(err))
+		}
+	}
+	return err.Err
+}
+
+func verboseClientError(aerr *client.APIError) string {
```

Comment:
Love these functions! It is gonna help our future customer support department a lot (and therefore ourselves).